panel/widgets/tables.py

```diff
@@ -476,8 +476,11 @@ def _process_column(self, values):
         return values
 
     def _get_data(self) -> Tuple[DataFrameType, DataDict]:
+        return self._get_data_any_df(self.value)
+
+    def _get_data_any_df(self, df: DataFrameType) -> Tuple[DataFrameType, DataDict]:
```

Comment:
Not sure I love the name of this method, something like `_convert_df_to_cds_data` seems better but would be happy with other suggestions.

Reply:
Yeah, I agree the name could be (much) better. But maybe make the name even more declarative because the function also filters and aligns multi-index the dataframe, maybe `_process_df_and_convert_to_cds`?

Reply:
Sounds good!